modules/velero-obtaining-by-downloading.adoc

```diff
@@ -23,16 +23,15 @@ The page includes instructions for:
 
 . Open a browser and navigate to link:https://{velero-domain}/docs/v{velero-version}/basic-install/#install-the-cli["Install the CLI" on the Verleo website].
 . Follow the appropriate procedure for macOS, GitHub, or Windows.
-. Download the Velero version appropriate for your version of OADP, according to the table that follows:
+. Download the Velero version appropriate for your version of OADP and {OCP} (OCP), according to the table that follows:
```

Comment:
The {OCP} is not rendering in the preview. Should it? I think you need an additional `include::_attributes/...` line, either to `attributes-microshift.adoc` or `attributes-openshift-dedicated` it appears.
I don't see any other instances of {OCP} (OCP) elsewhere in the docs. 

On second thought, I think this should be {product-title}, as this module [appears in the OKD docs](https://docs.okd.io/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html#velero-obtaining-by-downloading_oadp-troubleshooting).

Reply:
@mburke5678 From what i understand from https://issues.redhat.com/browse/OADP-773, we want this PR be part of the  OCP documentation as usual -- did I build the file structure incorrectly? From your comments it sounds like it will appear only in OKD docs.